pkg/cloudprovider/aws/fleet/eks.go

```diff
@@ -0,0 +1,28 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fleet
+
+import (
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+type EKSProvider struct {
+	coreV1Client *corev1.CoreV1Client
+}
+
+func (p *EKSProvider) Version() (string, error) {
+
+	return "1.18", nil
```

Comment:
I think we want to do an equivalent of `kubectl get --raw /version` instead of relying on EKS